systest/scripts/tempest_tests.sh

```diff
@@ -15,20 +15,23 @@
 # limitations under the License.
 #
 
-set -ex
+set -x
```

Comment:
Yes. For example in the current execution of the API tests there are 48 failures. If we allowed the tox target failure (which is when any test fails) to abort we would never run any downstream tests like scenario because this bash script would exit right away.